### PR TITLE
added home-assistant

### PR DESCRIPTION
**Pull request by chriskacerguis**
Added Home Assistant version 0.50.2 to the catalog.

### CODE REVIEW

File: templates/home-assistant/0/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  home-assistant:
+    image: homeassistant/home-assistant:0.51.2
+    privileged: true
```
Comment (rawmind0) [inline templates/home-assistant/0/docker-compose.yml]: I take a look at https://home-assistant.io/docs/installation/docker/, Is privileged mode mandatory??

**Comment by rawmind0** [comment]
Hi @chriskacerguis , thanks so much for the PR.

Is privileged mode mandatory??

File: templates/home-assistant/0/docker-compose.yml (hunk 1 above)
Comment (chriskacerguis) [reply templates/home-assistant/0/docker-compose.yml]: Well, yes and no.  If you are using Z-wave / Insteon / etc for automation those are devices that are plugged into the host machine.  So, IF you are using those (which most people are) then yes (that said there are work arounds, but they aren't for the faint of heart)

Comment (rawmind0) [reply templates/home-assistant/0/docker-compose.yml]: If it isn't mandatory, what do you think to add a question in rancher-compose?? :)
```
-    privileged: true
+    privileged: ${privileged}
```

File: templates/home-assistant/0/rancher-compose.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: '2'
+catalog:
+  name: "Home Assistant"
+  version: "0.50.2"
+  description: "Home Assistant is an open-source home automation platform running on Python 3"
```
Comment (rawmind0) [inline templates/home-assistant/0/rancher-compose.yml]: What do you think to add a question??
```
+    - variable: "privileged"
+      description: | 
+        Enable/disable to run in privileged mode.
+      label: "Privileged mode:"
+      required: true
+      default: false 
+      type: "boolean"
```

Comment (chriskacerguis) [reply templates/home-assistant/0/rancher-compose.yml]: Good idea!  I'll update the PR ASAP.

**Comment by chriskacerguis** [comment]
Going to close this and open another.

**Comment by chriskacerguis** [comment]
@rawmind0 new PR here -> https://github.com/rancher/community-catalog/pull/706

**Pull request closed without merging**